src/AgGrid.fs

```diff
@@ -4,18 +4,200 @@ module Feliz.AgGrid
 open System
 open Fable.Core
 open Fable.Core.JsInterop
-
 open Feliz
 
 let agGrid : obj = import "AgGridReact" "ag-grid-react"
 
-importAll "ag-grid-community/styles/ag-grid.css"
-importAll "ag-grid-community/styles/ag-theme-alpine.css"
-importAll "ag-grid-community/styles/ag-theme-balham.css"
-importAll "ag-grid-community/styles/ag-theme-material.css"
+// User should load the CSS files in their own project
+//importAll "ag-grid-community/styles/ag-grid.css"
+//importAll "ag-grid-community/styles/ag-theme-alpine.css"
+//importAll "ag-grid-community/styles/ag-theme-balham.css"
+//importAll "ag-grid-community/styles/ag-theme-material.css"
+
+// https://www.ag-grid.com/javascript-data-grid/row-object/
+[<Erase>]
+type IRowNode<'row> = {
+    id: string
+    data : 'row
+    updateData: 'row -> unit
+    setData: 'row -> unit
+    setSelected : bool -> unit
+    rowIndex : int
+    rowTop : int
+    displayed : bool
+    isHovered : bool
+    isFullWidthCell : bool
+    isSelected : bool
+}
+
+[<Erase>]
+type ICellRange = {
+    id : string
+    startRow : obj
+    endRow : obj
+}
+    with 

```

Comment:
I'm not certain that this is our usual formatting style. Maybe we should get fantomas set up on this codebase and run it?